### PR TITLE
Show Ack Modal when user click Create Pool/Add Liquidity on EthPoW

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
On Ethereum PoW, show Ack Modal when user clicks (both desktop and mobile):
- Create Pool button
- Add liquidity button

Desktop | Mobile
--- | ---
image | image

### CODE REVIEW

File: src/pages/Pools/EthPowAckModalContext.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { useContext } from 'react'
+
+export const EthPowAckModalContext = React.createContext<[string, (t: string) => void]>([
+  '',
+  () => {
+    // empty
```
[the remark this answers is not in the document]
Comment (nguyenhuudungz) [reply src/pages/Pools/EthPowAckModalContext.tsx:6]: Why need context here?

File: src/pages/ProAmmPools/CardItem.tsx [hunk 2]
```diff
@@ -70,6 +72,9 @@ export default function ProAmmPoolCardItem({ pair, onShared, userPositions, idx
   const { chainId } = useActiveWeb3React()
   const theme = useTheme()
   const [isOpen, setIsOpen] = useState(true)
+  const history = useHistory()
+  const [_, setUrlOnEthPoWAck] = useEthPowAckModalContext()
```
Comment (namgold) [inline src/pages/ProAmmPools/CardItem.tsx:76]:
```suggestion
  const [, setUrlOnEthPoWAck] = useEthPowAckModalContext()
```

File: src/pages/ProAmmPools/ListItem.tsx [hunk 3]
```diff
@@ -94,6 +96,9 @@ export default function ProAmmPoolListItem({ pair, idx, onShared, userPositions,
   const { chainId } = useActiveWeb3React()
   const theme = useTheme()
   const [isOpen, setIsOpen] = useState(pair.length > 1 ? idx === 0 : false)
+  const history = useHistory()
+  const [_, setUrlOnEthPoWAck] = useEthPowAckModalContext()
```
Comment (namgold) [inline src/pages/ProAmmPools/ListItem.tsx:100]:
```suggestion
  const [, setUrlOnEthPoWAck] = useEthPowAckModalContext()
```

File: src/pages/Pools/EthPowAckModalContext.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/pages/Pools/EthPowAckModalContext.tsx:6]: Without this, ESLint complains....

File: src/pages/ProAmmPools/CardItem.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/pages/ProAmmPools/CardItem.tsx:76]: Good to know this!

File: src/pages/Pools/EthPowAckModalContext.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (namgold) [reply src/pages/Pools/EthPowAckModalContext.tsx:6]: Why not use redux?

[the remark this answers is not in the document]
Comment (hungdoansy) [reply src/pages/Pools/EthPowAckModalContext.tsx]: Used Redux now

**Review by nguyenhuudungz — APPROVED** [review]

**Review by namgold — APPROVED** [review]

**Pull request merged**